kitchen-tests/Berksfile

```diff
@@ -1,12 +1,8 @@
 source "https://supermarket.chef.io"
 
-#cookbook "webapp", path: "cookbooks/webapp"
+cookbook "webapp", path: "cookbooks/webapp"
```

Comment:
it'd probably be more useful to focus on getting the learnchef tutorial cookbooks working again (below) and i think those are fairly "webapp"-ey and would provide similar coverage and validate the tutorials again.

Reply:
Yep I was going to get those working as well. This is what happens when I go in to fix a simple failure and get sucked in

Reply:
So to be a bit more clear -- the direction I was going was to replace the webapp cookbook with those learnchef tutorial cookbooks since those exercise mysql+httpd cookbooks and they're maintained.  I just never got around to deleting the webapp cookbook, which was the plan...